addons/mail/static/src/components/emoji_picker/emoji_picker.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.EmojiPicker" owl="1">
+        <div class="o_EmojiPicker" t-attf-class="{{ className }}" t-ref="root">
+            <EmojiCategoryBar emojiCategoryViews="props.record.emojiCategoryViews"/>
+            <EmojiShowcase emojiViews="props.record.emojiViews"/>
```

Comment:
You should pass `EmojiShowcase` record to this component. Have a 1:1 map between a Component and Model.